policybot/pkg/testflakes/testflakes.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
```

Comment:
As mentioned before, please make this a Go-style comment instead of a Java-style comment.

And remove the blank line after the comment so that it is right against the package statement. This will make the comment the official package documentation and will show up on godoc.org